app/cdap/components/Connections/Create/index.tsx

```diff
@@ -237,7 +235,7 @@ export function CreateConnection({
 
   return (
     <div className={classes.root}>
-      <If condition={!hadInitial && state.activeStep === ICreateConnectionSteps.CONNECTOR_CONFIG}>
+      <If condition={state.activeStep === ICreateConnectionSteps.CONNECTOR_CONFIG}>
```

Comment:
I know this is existing code but i think we should update things when we edit them (remove if component)

Reply:
Done